--- certora/munged/src/StaticATokenLM.sol	2023-04-23 15:35:33
+++ certora/tests/certora/StaticATokenLM.sol3.sol	2023-04-23 17:49:12
@@ -505,7 +505,8 @@
     uint256 shares = staticAmount;
 
     if (staticAmount > 0) {
-      amountToWithdraw = previewRedeem(staticAmount);
+      /// AssignmentMutation of: amountToWithdraw = previewRedeem(staticAmount);
+      amountToWithdraw = 0;
     } else {
       shares = previewWithdraw(dynamicAmount);
     }
